spacemesh0.1.md

```diff
@@ -14,23 +14,23 @@ This guide walks you though and describes the features and the supported functio
 
 Here are some scenarios that are not yet supported in the Spacemesh 0.1 Testnet release:
 
-1. Join the Testnet from a Node and a CLI wallet built directly from source code.
-2. Get Smesh coins from a tap.
+1. Join the Testnet from a Node and a CLI wallet built directly from source code. 
+2. Get Smesh coins from a tap account.
 3. Browse the Testnet using a block explorer and a dashboard.
 4. Run the App in Wallet-only mode without a Node.
 5. Run a node without producing blocks.
 
-We plan to introduce these roll out software to support these use cases in subsequent Spacemesh Testnet releases throughout 2020.
+We plan to introduce these roll out software to support these use cases in subsequent Spacemesh Testnet releases throughout 2020. 
 
 ## Spacemesh 0.1 Cheat Sheet
 
-- Network id: 0.
-- POST init size per miner:	~20 minutes on a modern CPU.
-- Layer mining award: 50 SMCH.
-- Awards maturity: 4 layers (20 minutes).
-- Non-routable Nodes are not supported.
-- Layer duration: 5	minutes.
-- Number of layers in an epoch:	288.
-- Epoch duration: 24 hours.
-- Coin transfer transaction fee: 1-3 Smetoshis.
-- Time to first award: ~49 hours.
+- Network id: 0
+- POST init size per miner:	TBD (about 20 minutes on a modern CPU)
+- Layer mining award: 50 SMCH
+- Award maturity: After 4 fully conifermed layers (about 20 minutes)
+- Non-routable Nodes are not supported
+- Layer duration: 5	minutes
+- Number of layers in an epoch:	288
+- Epoch duration: 24 hours
+- Coin transfer transaction fee: 1-3 Smetoshis
+- Time to first award: 24-72 hours
```

Comment:
This page has two sections with bullet points. Either both or none should have a period in the end of each sentence. I removed all periods here.